docs/resources/Voice.md

```diff
@@ -53,3 +53,54 @@ Used to represent a user's voice connection status.
 ## List Voice Regions % GET /voice/regions
 
 Returns an array of [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) objects that can be used when setting a voice or stage channel's [`rtc_region`](#DOCS_RESOURCES_CHANNEL/channel-object-channel-structure).
+
+## Get Current User Voice State % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/voice-states/@me
+
+Returns the current user's [voice state](#DOCS_RESOURCES_VOICE/voice-state-object) in the guild.
+
+## Get User Voice State % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/voice-states/@me
```

Comment:
isn't this missing the user ID in the path?

Reply:
oops ty